Persistent Work folder

agowa338 opened the pull request:
Make the work folder persistent

brasmith-ms commented:
Hi @agowa338, we had considered adding a persistent working directory at first but the log files generated by FAH aren't really needed by most people so it's better to just have them evaporate when Sandbox is closed. I think it would be a good idea to add a flag to the install script that enables a persistent working directory for those who want it.

There are a couple issues with the changes here: first the script does not work if it's been run with an older version (the mapped folder is not created, so Sandbox can't find it), and second you added powershell syntax to the part of the script that creates the .cmd logon file. These are minor, and will be resolved once we switch to adding a working directory as an optional feature.

If you want to make a patch for this go ahead, otherwise I'll add those in when I get a free moment.

agowa338 commented:
The work directory also holds the current working unit, therefore it is good to have it persistent, so that the already overloaded scheduling servers do not have to schedule you multiple times just because you closed and reopened the client or restarted the computer ;-)

I added a switch to use no persistent directory.
I also removed the change to the cmd part, that was by accident.
